Update to TFC 0.79.24

dexman545 opened the pull request.

dexman545 committed:
Update to TFC 0.79.24

vidaj commented:
Hi!

I'm sorry, but I can't accept this PR. You have changed mcmod.info and build.gradle, so this won't compile or run as a proper mod anymore. Also, you have commited eclipse binary files and launch configurations. I don't believe this repo needs to contain fml and forge credits/license files.

If you fix those issues, I will accept the PR. (The only change to mcmod.info should be adding yourself to authors and/or credits, and the only change to build.gradle should be the forge-bump).

dexman545 committed:
Removed Unnessary files.
Fixed mcmod.info

dexman545 commented:
I've fixed it now. Sorry about that, I don't actually program, so I was unsure about github pushes for code repos. I also think I've fixed the files for your water compatibility mod.
